services/github/github-last-commit.service.js

```diff
@@ -15,14 +15,26 @@ const schema = Joi.array()
         author: Joi.object({
           date: Joi.string().required(),
         }).required(),
+        committer: Joi.object({
+          date: Joi.string().required(),
+        }).required(),
       }).required(),
     }).required()
   )
   .required()
 
+const queryParamSchema = Joi.object({
+  by_committer: Joi.equal(''),
```

Comment:
Similar to https://github.com/badges/shields/pull/8543#discussion_r998655480 I'd make this `display_timestamp` with options `author` and `committer` defaulting to `author`

Reply:
Good point, thanks. Updated